### PR TITLE
add s3path

### PR DESCRIPTION
**Pull request by liormizr**
See https://pypi.org/project/s3path

### CODE REVIEW

File: recipes/s3path/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 71a00b7714d0555d3588a46db20fcda9c1f1cb611232ee5563e89e291e3ae7a1
+
+build:
+  number: 0
+  noarch: python
+  skip: True  # [py<35]
```
Comment (npavlovikj) [inline recipes/s3path/meta.yaml]: Please remove the line `skip: True`.

File: recipes/s3path/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
```
Comment (npavlovikj) [inline recipes/s3path/meta.yaml]: Add `python >=3.5` instead.

File: recipes/s3path/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  run:
+    - python
```
Comment (npavlovikj) [inline recipes/s3path/meta.yaml]: Add `python >=3.5` instead.

File: recipes/s3path/meta.yaml (hunk 2 above)
Comment (liormizr) [reply recipes/s3path/meta.yaml]: very cool, Tx!

**Comment by npavlovikj** [comment]
This is ready for merge/review @conda-forge/staged-recipes , thanks!

**Comment by beckermr** [comment]
Looks great!

**Comment by liormizr** [comment]
tx guys

**Pull request merged**